changelog.json

```
@@ -1,5 +1,23 @@
 {
   "releases": {
+    "3.2.1-beta2": [
+      "[Added] Add new RStudio bundle identifier - #16339",
```

I think this should be a fix (and moved down with the rest of fixes):
```suggestion
      "[Fixed] Fix support of newer versions of RStudio - #16339",
```